tests/Unit/Evolution/Systems/GeneralizedHarmonic/Test_Constraints.cpp

```diff
@@ -931,3 +1211,21 @@ SPECTRE_TEST_CASE("Unit.Evolution.Systems.GeneralizedHarmonic.ConstraintEnergy",
   test_constraint_energy_random<3, Frame::Inertial, double>(
       std::numeric_limits<double>::signaling_NaN());
 }
+
+SPECTRE_TEST_CASE(
+    "Unit.Evolution.Systems.GeneralizedHarmonic.ConstraintComputeTags",
+    "[Unit][Evolution]") {
+  pypp::SetupLocalPythonEnvironment local_python_env{
+      "Evolution/Systems/GeneralizedHarmonic/"};
+  // Test the F constraint against Kerr Schild
```

Comment:
ping @carmaza - how do I avoid this test from timing out?

Reply:
```
// [[TimeOut, 5]]
SPECTRE_TEST_CASE(
    "Unit.Evolution.Systems.GeneralizedHarmonic.ConstraintComputeTags",
    "[Unit][Evolution]") {
```

where 5 is the time in seconds for a timeout.